e2e/go.mod

```diff
@@ -11,8 +11,8 @@ require (
 	github.com/jackc/pgtype v1.12.0
 	github.com/jackc/pgx/v4 v4.17.2
 	github.com/stretchr/testify v1.8.1
-	golang.org/x/tools v0.1.12
-	google.golang.org/grpc v1.50.1
+	golang.org/x/tools v0.3.0
+	google.golang.org/grpc v1.51.0
```

Comment:
wait, what happened here, we unupgraded grpc?

Reply:
No. I had a typo and fixed it in a later commit

Reply:
my bad, brain fart